modules/tasks/tasks/src/test/java/org/motechproject/tasks/web/SettingsControllerTest.java

```diff
@@ -37,30 +41,38 @@ public void setUp() {
     @Test
     public void testGetSettings() {
         when(settingsFacade.getProperty(TASK_POSSIBLE_ERRORS_KEY)).thenReturn(TASK_POSSIBLE_ERRORS_VALUE);
+        when(settingsFacade.getProperty(TASK_LOG_ACTIVITIES_KEY)).thenReturn(TASK_LOG_ACTIVITIES_VALUE);
 
         SettingsDto dto = controller.getSettings();
 
-        verify(settingsFacade, times(1)).getProperty(anyString());
+        verify(settingsFacade, times(1)).getProperty(TASK_POSSIBLE_ERRORS_KEY);
         assertEquals(TASK_POSSIBLE_ERRORS_VALUE, dto.getTaskPossibleErrors());
+
+        verify(settingsFacade, times(1)).getProperty(TASK_LOG_ACTIVITIES_KEY);
+        assertEquals(TASK_LOG_ACTIVITIES_VALUE, dto.getTaskLogActivities());
     }
 
     @Test
     public void testSaveSettings() throws BundleException {
         SettingsDto dto = new SettingsDto();
         dto.setTaskPossibleErrors(TASK_POSSIBLE_ERRORS_VALUE);
+        dto.setTaskLogActivities(TASK_LOG_ACTIVITIES_VALUE);
 
         controller.saveSettings(dto);
 
-        verify(settingsFacade, times(1)).setProperty(anyString(), anyString());
+        verify(settingsFacade, times(1)).setProperty(eq(TASK_LOG_ACTIVITIES_KEY), anyString());
+        verify(settingsFacade, times(1)).setProperty(eq(TASK_POSSIBLE_ERRORS_KEY), anyString());
```

Comment:
We can predict second parameter in these setters. It should be "TASK_LOG_ACTIVITES_VALUE" and "TASK_POSSIBLE_ERRORS_VALUE"